src/olympia/devhub/tests/test_views_validation.py

```diff
@@ -63,6 +63,16 @@ def test_upload_processed_validation(self):
         upload = FileUpload.objects.get(uuid=uuid)
         assert upload.processed_validation['errors'] == 1
 
+    def test_login_required(self):
```

Comment:
It tests that if there's a user_id on the object, then a login is required. It later does a logout and confirms that it gets redirected.